packages/common/src/content/edit.ts

```diff
@@ -86,7 +91,8 @@ export async function updateContent(
   // let resources;
 
   // get the backing item & data
-  const model = await getModel(content.id, requestOptions);
+  const item = await getItem(content.id, requestOptions);
```

Comment:
@abp6318 we probably want a similar comment here about not calling `getModel()` b/c some content types have non-JSON data